src/plugins/wb-disable/disable.js

```diff
@@ -30,15 +30,29 @@ var componentName = "wb-disable",
 			$html = wb.html,
 			i18n = wb.i18n,
 			pageUrl = wb.pageUrlParts,
-			li, param;
+			li, param,
+			i18nTemporary = {
+				en: {
+					"disable-notice-h": "Notice: Basic HTML",
+					"disable-notice": "You are viewing Basic HTML view. Some features may be disabled."
+				},
+				fr: {
+					"disable-notice-h": "Avis: Version HTML simplifiée",
+					"disable-notice": "Vous naviguez présentement sur la version HTML simplifiée de cette page. Certaine fonctionalités peuvent être déactivées."
+				}
+			},
+			noticeHeader = i18n( "disable-notice-h" ),
```

Comment:
@LaurentGoderre then what about the WCAG language of part issue? Should be ignored for this PR when we have a way to avoid it?